apps/console/src/features/branding/models/custom-text-preference.ts

```diff
@@ -86,3 +86,23 @@ export enum CustomTextConfigurationModes {
      */
     TEXT_FIELDS = "TEXT_FIELDS"
 }
+
+/**
+ * Interface for the custom text preference screen meta.
+ */
+export interface CustomTextPreferenceScreenMetaInterface {
+    [key: string]: {
+        /**
+         * Is the text preference editable.
+         */
+        EDITABLE: boolean;
+        /**
+         * Screen name of the text preference.
+         */
+        SCREEN: string;
+        /**
+         * Is the text preference multi-line.
+         */
+        MULTI_LINE: boolean;
```

Comment:
Since these are not constants and not members of an enum, let's keep the property names in lower case.

Reply:
These are coming from the meta values in the .properties file as these values, hence we cannot change them IMO.

Reply:
Ack